src/libflac/bitreader.c

```diff
@@ -30,6 +30,14 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#ifndef REGISTER
```

Comment:
`register` is a hint that's ignored most times, anyway, so I think I would just drop the whole modifier everywhere.  GCC is smart enough, and we have ~12 open registers on the 8266 to work with, anyway...

Reply:
I agree but I didn't try the effect on gcc-4.8.2 (and whatever version is used with esp32)